api/src/main/java/org/openmrs/util/TimeZoneUtil.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+import org.openmrs.User;
+import org.openmrs.api.context.Context;
+import org.openmrs.ui.framework.UiFrameworkConstants;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+import static org.joda.time.DateTimeZone.UTC;
+
+/**
+ * Helps provide tools to support recommended OpenMRS time zones conventions.
+ * 
+ * @see https://wiki.openmrs.org/display/docs/Time+Zones+Conventions
+ */
+public class TimeZoneUtil {
+	
+	/**
+	 * Convert a date to the client timezone, and format it, to be readable to for the user. .
+	 * 
+	 * @param date The date.
+	 * @param format the format to be used on the date
+	 * @return string with the date on the client timezone, formatted and ready to be displayed.
+	 */
+	public static String toTimezone(Date date, String format) {
+		String clientTimezone = Context.getAuthenticatedUser().getUserProperty("clientTimezone");
+		return toTimezone(date, format, clientTimezone);
+		
+	}
+	
+	/**
+	 * Formats a date while expressing it in the specified timezone.
+	 * 
+	 * @param date The date.
+	 * @param format the format to be used on the date
+	 * @param timezone The tz database name, eg. "Europe/Zurich", if for some reason that param is null,
+	 *            it will use UTC.
+	 * @return string with the date on the client timezone, formatted and ready to be displayed.
+	 */
+	public static String toTimezone(Date date, String format, String timezone) {
+		if (StringUtils.isEmpty(timezone)) {
+			timezone = "UTC";
```

Comment:
Do you wanna define a constant for `UTC`?

Reply:
Do you think is necessary? I only use it here.

Reply:
I replaces that "UTC" by `import static org.joda.time.DateTimeZone.UTC`